flytekit/core/type_engine.py

```diff
@@ -1532,7 +1532,6 @@ async def _literal_map_to_kwargs(
             raise TypeTransformerFailedError(
                 f"Error converting input '{k}' at position {i}:\n"
                 f"Literal value: {lm.literals[k]}\n"
-                f"Literal type: {literal_types}\n"
```

Comment:
We can remove it since it's None in most cases, and the Python type should be sufficient